tests/php/sync/test_class.jetpack-sync-actions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class WP_Test_Jetpack_Sync_Actions extends WP_UnitTestCase {
+	function test_get_sync_status() {
+		$no_checksum = Jetpack_Sync_Actions::get_sync_status();
+		$this->assertArrayNotHasKey( 'posts_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comments_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'post_meta_checksum', $no_checksum );
+		$this->assertArrayNotHasKey( 'comment_meta_checksum', $no_checksum );
+
+		$kitchen_sink_checksum = Jetpack_Sync_Actions::get_sync_status(
```

Comment:
what do you think about one test per `Jetpack_Sync_Actions::get_sync_status` call?

Reply:
Multiple assertions in a single test is something I've considered to be fine until now.

As stated before, it's just testing a single concept. I'd love to have a simpler way to write this while maintaining human-readability, but PHPUnit doesn't seem to allow it.